ivy/functional/backends/paddle/elementwise.py

```diff
@@ -1,15 +1,16 @@
 # global
-from typing import Union, Optional
-
 import math
+from typing import Optional, Union
+
 import paddle
-import ivy.functional.backends.paddle as paddle_backend
+
 import ivy
+import ivy.functional.backends.paddle as paddle_backend
 from ivy import promote_types_of_inputs
 from ivy.func_wrapper import (
-    with_unsupported_device_and_dtypes,
     with_supported_device_and_dtypes,
```

Comment:
```suggestion
    with_supported_device_and_dtypes, you don't need to change this.
```

Reply:
Hey @samthakur587, the changes are simply reordering the functions imported in ascending order, which was automatically done by `black`. No imports are removed or added. Please let me know if you want me to revert the imports to their initial state.

Reply:
can you please just update your branch to latest commit. i want to check it's still coming or not. just for review. 😄 
```suggestion
    with_supported_device_and_dtypes,
```

Reply:
Hello @samthakur587, I updated my branch to the latest commit. I'll now make the required changes you requested in subsequent messages.